website/docs/tutorials/introduction/00_tutorial_intro.md

```diff
@@ -0,0 +1,96 @@
+---
+id: tutorial-introduction
+title: 00 - Introduction
+---
+# Kivy Tutorial Introduction
+
+This is a three part tutorial designed to walk you throuhg the process of
+ devleoping your custom kivy application on the brain. We support two main user
+ interface development environments, ReactJS and Kivy.
+
+Part 1 will walk you through cloneing a template repository onto the
+brain. [**Amiga-app-template-kivy**](https://github.com/farm-ng/amiga-app-template-kivy/tree/main)
+ includes all the files required to begin developing a custom application.
+
+Part 2 ([**camera-streamer-kivy**](https://github.com/farm-ng/camera-streamer-kivy))
+will walk you through building a simple kivy interface to allow you to visualize the
+ oak camera streams from the brain.
+
+Part 3 ([**virtual-joystick**](https://github.com/farm-ng/virtual-joystick-v2)) will
+walk you through a more complicated kivy interface, sending movement controls and
+customizing app requirements.
```

Comment:
This content should instead be in either or both of the examples overview page and/or a (new) tutorials overview page. The intro to a given tutorial should be about it and any prereqs, not future tutorials. In the past, this was on the Examples Index page under a tutorials section.